jormungandr-integration-tests/src/mock/proto/node.rs

```diff
@@ -2678,7 +2678,7 @@ static file_descriptor_proto_data: &'static [u8] = b"\
     \x08announce\x18\x01\x20\x01(\x0b2\x17.iohk.chain.node.HeaderH\0R\x08ann\
     ounce\x125\n\x07solicit\x18\x02\x20\x01(\x0b2\x19.iohk.chain.node.BlockI\
     dsH\0R\x07solicit\x12?\n\x07missing\x18\x03\x20\x01(\x0b2#.iohk.chain.no\
-    de.PullHeadersRequestH\0R\x07missingB\x06\n\x04item2\xb3\x07\n\x04Node\
+    de.PullHeadersRequestH\0R\x07missingB\x06\n\x04item2\xb4\x07\n\x04Node\
```

Comment:
This generated crap in git is precisely why it's better to invoke protoc in the build script.